HHS/OPRE-OPS
# feat: ✨ adds CAN Transfer filter

## What changed

- adds CAN Transfer filter

## Issue

#347 #2892

## How to test

1. goto CANS list
2. apply Transfer filter
3. see CANs Table update
4. 💰 profit

## Screenshots

https://github.com/user-attachments/assets/e26bc8a5-cce5-4935-89b5-0a5b7b4228ba

## Definition of Done Checklist
